examples/hello-world/step-by-step/higgs/sklearn-linear/sklearn_linear.ipynb

```diff
@@ -92,11 +92,7 @@
     "### Data Cleaning \n",
     "\n",
     "We noticed from time-to-time the Higgs dataset is making small changes which causing job to fail. so we need to do some clean up or skip certain rows. \n",
-    "For example: certain floating number mistakenly add an alphabetical letter at some point of time. This may have already fixed by UCI. \n",
-    "\n",
-    "As of today: 2023-11-21, the dataset we downloaded seems to have some value that can't convert to floating point. value such as \"0.000000000000000000e+00.1\"\n",
-    "\n",
-    "Here we notice this seems to be at 1st row of the files. so we are going to use skip_rows during data loading to avoid this. \n"
```

Comment:
maybe the cleaning data part should be moved instead into the "prepare_data" notebook?

Reply:
this part is on-the-fly operation during execution, so better to keep it here I think? @chesterxgchen